src/test/app/PermissionedDomains_test.cpp

```diff
@@ -77,6 +78,22 @@ class PermissionedDomains_test : public beast::unit_test::suite
         env(pdomain::deleteTx(alice, domain));
     }
 
+    // Verify that PD cannot be created or updated if credentials are disabled
+    void
+    testCredentialsDisabled()
```

Comment:
nit: would prefer if this was under `testDisabled`.

Reply:
the reason why this is separate is because the normal test pattern is to use one `FeatureBitset` per test (or per test run). I like sticking to patterns, makes things more predictable.